### PR TITLE
Fix # : Lowfi Landscape: OngoingTopicListActivity

### PR DESCRIPTION
**Pull request by virajpatel2411**
## Mock:
Mock: https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/ce21cbf2-0439-47d7-bb37-005cfc6b9742/UP-Topics-in-Progress-

## Explanation:
Landscape res of Ongoingtopicfragment created
Issue #791 
@rt4914 @veena14cs @nikitamarysolomanpvt Can anyone please check the UI as per mock so that if it is correct then after that I can add test case?

### CODE REVIEW

**Comment by nikitamarysolomanpvt** [comment]
@virajpatel2411 Please add test cases then assign back to me.

**Comment by virajpatel2411** [comment]
@veena14cs @nikitamarysolomanpvt I have added the test cases. Please review it. @rt4914

File: app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt [hunk 1]
```diff
@@ -132,13 +138,33 @@ class OngoingTopicListActivityTest {
...
+  /////////////////////////////////////////////////////////////////////////////////////////////////////////
+
   private fun createOngoingTopicListActivityIntent(internalProfileId: Int): Intent {
     return OngoingTopicListActivity.createOngoingTopicListActivityIntent(
       ApplicationProvider.getApplicationContext(),
       internalProfileId
     )
   }
 
+
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt:165]: Remove these 3 blank unnecessary lines.

File: app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt [hunk 2]
```diff
@@ -132,13 +138,33 @@ class OngoingTopicListActivityTest {
...
+        ViewActions.click())
+      onView(withId(R.id.topic_name_text_view)).check(matches(withText(R.string.topic_name_text_view)))
+      onView(withId(R.id.story_count_text_view)).check(matches(withText(R.string.story_count_text_view)))
+      onView(ViewMatchers.isRoot()).perform(OrientationChangeAction.orientationLandscape())
+      onView(withId(R.id.topic_name_text_view)).check(matches(withText(R.string.topic_name_text_view)))
+      onView(withId(R.id.topic_name_text_view)).check(matches(withText(R.string.story_count_text_view)))
+    }
+  }
+
+  /////////////////////////////////////////////////////////////////////////////////////////////////////////
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt:156]: Remove this comment and a blank line above this.

File: app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt [hunk 3]
```diff
@@ -132,13 +138,33 @@ class OngoingTopicListActivityTest {
     }
   }
 
+///////////////////////////////////////Test Case for Landscape Mode////////////////////////////////////////
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt:141]: Remove this comment and a blank line below this.

File: app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt [hunk 4]
```diff
@@ -132,13 +138,33 @@ class OngoingTopicListActivityTest {
...
+
+  @Test
+  fun testTopicPracticeFragment_loadFragment_changeOrientation_textIsCorrect() {
+    launch<OngoingTopicListActivity>(createOngoingTopicListActivityIntent(internalProfileId)).use {
+      onView(withId(R.id.ongoing_topic_list)).perform(
+        ViewActions.click())
+      onView(withId(R.id.topic_name_text_view)).check(matches(withText(R.string.topic_name_text_view)))
+      onView(withId(R.id.story_count_text_view)).check(matches(withText(R.string.story_count_text_view)))
+      onView(ViewMatchers.isRoot()).perform(OrientationChangeAction.orientationLandscape())
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt:150]: Nit update to make code more readable
`onView(isRoot()).perform(orientationLandscape())`

File: app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt [hunk 5]
```diff
@@ -132,13 +138,33 @@ class OngoingTopicListActivityTest {
     }
   }
 
+///////////////////////////////////////Test Case for Landscape Mode////////////////////////////////////////
+
+  @Test
+  fun testTopicPracticeFragment_loadFragment_changeOrientation_textIsCorrect() {
+    launch<OngoingTopicListActivity>(createOngoingTopicListActivityIntent(internalProfileId)).use {
+      onView(withId(R.id.ongoing_topic_list)).perform(
+        ViewActions.click())
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt:147]: Nit Update:
`onView(withId(R.id.ongoing_topic_list)).perform(click())`

File: app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt [hunk 6]
```diff
@@ -132,13 +138,33 @@ class OngoingTopicListActivityTest {
...
+      onView(withId(R.id.topic_name_text_view)).check(matches(withText(R.string.topic_name_text_view)))
+      onView(withId(R.id.story_count_text_view)).check(matches(withText(R.string.story_count_text_view)))
+      onView(ViewMatchers.isRoot()).perform(OrientationChangeAction.orientationLandscape())
+      onView(withId(R.id.topic_name_text_view)).check(matches(withText(R.string.topic_name_text_view)))
+      onView(withId(R.id.topic_name_text_view)).check(matches(withText(R.string.story_count_text_view)))
```
Comment (rt4914) [inline app/src/sharedTest/java/org/oppia/app/ongoingtopiclist/OngoingTopicListActivityTest.kt:152]: The ID of this case is incorrect.

**Review by rt4914 — CHANGES_REQUESTED** [review]
The implementation is correct but there is one major issue.

You have made changes on your `develop` branch. Instead you should create a new branch on your fork.
Follow these steps always: https://github.com/oppia/oppia-android/wiki#instructions-for-making-a-code-change

Now, do not add extra blank lines.

Also, the test case failing. Please always fun your test case and if they pass then only send it for review.

Your branch is not up-to-date with develop please update your code.

Your title does not have a issue number. Please add your issue number to your PR title as per the convention.

**Comment by virajpatel2411** [comment]
@rt4914 Closing this PR...will make a new apt PR and let you know in some time. Will I be assigned to another issue so while this PR's review is going on I can work on another one?

**Comment by rt4914** [comment]
> @rt4914 Closing this PR...will make a new apt PR and let you know in some time. Will I be assigned to another issue so while this PR's review is going on I can work on another one?

I suggest you focus on one PR at a time. Once you finish two PRs then we can think of multiple issues at a time.

**Comment by virajpatel2411** [comment]
> > @rt4914 Closing this PR...will make a new apt PR and let you know in some time. Will I be assigned to another issue so while this PR's review is going on I can work on another one?
> 
> I suggest you focus on one PR at a time. Once you finish two PRs then we can think of multiple issues at a time.

Okayy

**Pull request closed without merging**